File: Educational Codeforces Round 142 (Rated for Div. 2)/A. GamingForces.cpp
#include <bits/stdc++.h>
using namespace std;
 
int main() {
    int n,m,temp,count1=0;
    cin >> n;
    
    for (int i=0; i<n; i++) {
        cin >> m;
        count1=0;
        
        for (int j=0; j<m; j++) {
            cin >> temp;
            
            if (temp == 1) {
                count1 ++;
            }
        }
        
        cout << m - count1 + (count1+1)/2 << "\n";
    }
}

File: Educational Codeforces Round 142 (Rated for Div. 2)/A. GamingForces.py
import sys

for _ in range(int(sys.stdin.readline().strip())):
    m = int(sys.stdin.readline().strip())
    count1 = 0
    for temp in map(int, sys.stdin.readline().strip().split()):
        if temp == 1:
            count1 += 1
    print(m - count1 + (count1+1)//2)
